egs/mgb2/asr/local/mgb_data_prep.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2020 Kanari AI 
+# (Amir Hussein)
+
+
+if [ $# -ne 3 ]; then
+  echo "Usage: $0 <DB-dir> <process-xml>"
+  exit 1;
+fi
+mer=80
+db_dir=$1
+process_xml=$2
+subset=$3  # subset of training data
+test_dir=data/test
+
+train_dir=data/train
+dev_dir=data/dev
+
+for x in $train_dir $dev_dir $test_dir; do
+  mkdir -p $x
+#  if [ -f ${x}/wav.scp ]; then
+#    mkdir -p ${x}/.backup
+#    mv $x/{wav.scp,utt2spk,spk2utt,segments,text} ${x}/.backup
+#  fi
```

Comment:
I think these backup code is necessary, could you comment them?

Reply:
done